srcpkgs/youtube-dl/template

```diff
@@ -13,7 +13,7 @@ license="Unlicense"
 homepage="https://yt-dl.org"
 changelog="https://raw.githubusercontent.com/ytdl-org/youtube-dl/master/ChangeLog"
 distfiles="${homepage}/downloads/latest/${pkgname}-${version}.tar.gz"
```

Comment:
If the website deletes old versions, might make sense to move to GitHub or PyPi.

Reply:
Ah looks like youtube-dl was just updated a moment ago, so will be removing it from this commit